src/ResourceGraph/ResourceGraph/Cmdlets/SearchAzureRmGraph.cs

```diff
@@ -186,5 +238,32 @@ private IEnumerable<string> GetSubscriptions()
 
             return SubscriptionCache.GetSubscriptions(this.DefaultContext);
         }
+
+        /// <summary>
+        /// Gets list of detailed tenants 
+        /// </summary>
+        /// <returns></returns>
+        private IList<IDictionary<string, object>> ListTenants()
+        {
+            try
+            {
+                var profileClient = new RMProfileClient(AzureRmProfileProvider.Instance.GetProfile<AzureRmProfile>());
```

Comment:
@OlhaTkachenko you should not be referencing types from the Accounts project -- all authentication needs to be encapsulated inside of the Accounts project and not used by any other project. What is the goal for the below code?

Reply:
We are doing ARM call not using SDK. It is why we need to get authorization token. Please let me know if you have other ideas. I will add tag, maybe somebody will know